### PR TITLE
added a request generator

### LINKED ISSUE #89: GraphQL request generator
What has to be done:
Currently writing a GraphQL query is quite complex due to the structure of a query string.

Why it has to be done:
image

When the task is done:
When a request can be used like the following example
image

### PR DESCRIPTION
**Pull request by lfb0801**
should make sending requests easier and closes #89

### CODE REVIEW

File: src/services/GraphQLQueryGenerator.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+class GraphQLRG {
+    static generateBody(_method, _function, _queryData, _variables) {
```
Comment (larsvandriel) [inline src/services/GraphQLQueryGenerator.js]: Method is too big and very hard to follow.

Comment (lfb0801) [reply src/services/GraphQLQueryGenerator.js]: small brain, child

Comment (MindHawk) [reply src/services/GraphQLQueryGenerator.js]: report for bully
